webcompat/static/js/lib/issues.js

```diff
@@ -489,6 +490,23 @@ issues.MainView = Backbone.View.extend({
 
   handleKeyShortcuts: function() {
     Mousetrap.bind("mod+enter", _.bind(this.addNewComment, this));
+  },
+
+  // Auto expanding feature for comments
```

Comment:
Yes, I understand the value of code reusability. Can we make a global function that works for all the elements that has js-autoExpand class? Although, I have added a comment there too.